package.json

```diff
@@ -9,7 +9,6 @@
     "aa-sdk/*",
     "account-kit/*",
     "templates/*",
-    "examples/*",
```

Comment:
will likely undo this, but wanna see if isolating the demo app fixes the vercel issue